Chalenges/set/Set.js

```diff
@@ -0,0 +1,95 @@
+class Set {
+  constructor() {
+    this.set = {};
+  }
+  add(value) {
+    if(!this.set[value]) {
+      this.set[value] = value;
+    } else {
+      return 'this is already here try adding something else';
```

Comment:
you can `throw` an error here, but i wouldn't return an error message.

I would probably just do nothing if the value is already there